arcgis-ios-sdk-samples/Analysis/Statistical query (group and sort)/StatisticalQueryGroupAndSortViewController.swift

```diff
@@ -372,3 +362,74 @@ class StatisticalQueryGroupAndSortViewController: UIViewController, UITableViewD
         }
     }
 }
+
+class GroupSortQueryResultsViewController: UITableViewController {
+    
+    var statisticsQueryResult: AGSStatisticsQueryResult? {
+        didSet {
+            statisticRecords = statisticsQueryResult?.statisticRecordEnumerator().allObjects ?? []
+        }
+    }
+    
+    var selectedGroupByFieldNames: [String] = []
+    
+    var statisticRecords: [AGSStatisticRecord] = [] {
+        didSet {
+            tableView?.reloadData()
```

Comment:
Querying the `tableView` property will cause the view to load. I would suggest guarding `isViewLoaded` before this call. Same with line `419`.